packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1609,7 +1658,11 @@ class TextInput {
         (_currentConnection!._client as DeltaTextInputClient).updateEditingValueWithDeltas(deltas);
         break;
       case 'TextInputClient.performAction':
-        _currentConnection!._client.performAction(_toTextInputAction(args[1] as String));
+        if (args[1] as String == 'TextInputAction.commitContent') {
```

Comment:
I'm going to review the engine PR and double check that this is the best way to send this data.